site/src/main/mdoc/laika/versionInfo.json

```diff
@@ -0,0 +1,32 @@
+{
+  "versions": [
+    { "displayValue": "1", "pathSegment": "v1", "fallbackLink": "/docs/quickstart.html", "label": "Dev" },
+    { "displayValue": "0.23", "pathSegment": "v0.23", "fallbackLink": "/docs/quickstart.html", "label": "Stable" },
+    { "displayValue": "0.22", "pathSegment": "v0.22", "fallbackLink": "/docs/quickstart.html", "label": "Stable" },
+    { "displayValue": "0.21", "pathSegment": "v0.21", "fallbackLink": "/index.html", "label": "EOL" },
+    { "displayValue": "Help me choose...", "pathSegment": "", "fallbackLink": "/versions.html" }
+  ],
+  "linkTargets": [
+    { "path": "/docs/auth.html", "versions": ["v0.22", "v0.23", "v1"] },
+    { "path": "/docs/client.html", "versions": ["v0.22", "v0.23", "v1"] },
```

Comment:
Until the laika fix comes through, we'll need to keep this file in sync across all three branches. So long as we don't add any new http4s versions or versioned pages I think we should be okay 😆